tailwind.config.js

```diff
@@ -16,12 +16,28 @@ module.exports = {
         'post': '0.01rem'
       },
       fontFamily: {
-        'inter': ['Inter', 'sans-serif']
+        'inter': ['Inter', 'sans-serif'],
+        'lato': ['Lato', 'sans-serif']
       },
       colors: {
+        gray02: '#E0E0E0',
+        gray05: '#F5F5F5',
         gray06: '#828282',
         grayIcon: '#C4C4C4',
-        primaryBlue: '#2f80ed'
+        foundationBlue: '#0D5AA0',
+        primaryBlue: '#2f80ed',
+        humanId_blue: '#023B60',
+        cyan: '#00ADB5',
+        cyan10: '#00ADD5'
+      },
+      boxShadow: {
+        '3xl': '-1.5px 3px 3px 0px rgba(0, 0, 0, 0.50);'
+      },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The value for the boxShadow property seems to have a semicolon at the end which is not needed in this context and might cause issues. Please remove it.

```diff
-        '3xl': '-1.5px 3px 3px 0px rgba(0, 0, 0, 0.50);'
+        '3xl': '-1.5px 3px 3px 0px rgba(0, 0, 0, 0.50)'
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->